includes/class-wc-order-item-coupon.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * Order Line Item (coupon).
+ *
+ * @class 		WC_Order_Item_Coupon
+ * @version		2.6.0
```

Comment:
Spacing is off on this docblock and a few of the other docblocks for the other order-* classes.